paper.tex

```diff
@@ -856,14 +856,11 @@ \section*{Genome ARGs}
 \caption{\label{fig-arg-in-pedigree}
 % TODO caption needs another pass. One the full section has been written
 % it'll be clearer what we need to say in the caption vs the text.
-The Ancestral Recombination Graph defined as arbitrary inheritance
-of genomes between diploid (or any ploidy; see text) individuals.
-(A) Each individual consists of two genomes, and each genome is
-inherited as a (potentially) recombinant mixture of the individual's parent's
-genomes. (B) Genomes are modelled as nodes in a graph, and the edges
-denote inheritance. Each edge is annotated with genome coordinates
-defining the specific region(s) inherited.
-(C) This description of inheritance fully defines the
+The Ancestral Recombination Graph defined in terms of genomes embedded
+in a pedigree. (a) Diploid individuals, visualised in a highly inbred pedigree and labelled $i_1$ to $i_8$, contain both paternal (left) and maternal (right) genomes labelled \textsf{A} to \textsf{P}. Black lines show transmission paths connecting genomes in the current generation (\textsf{A} to \textsf{D}) with their ancestors. Two recombination breakpoints, at positions 3 and 6 mean that genomes \textsf{A} and \textsf{C} are independent recombinants of the paternal genomes \textsf{E} and \textsf{F}: the region of genome transmitted through each edge is shown in these cases. Where genomes merge into a common ancestor, the merged region is shown darker, until all genomic regions are fully coalesced in genome \textsf{N} (b) The equivalent genome ARG, where there is a single type of node (representing a genome) along with annotations on all edges; annotations showing partial transmission are highlighted in bold. Recombination, such that causing the breakpoint at position 3, is associated with two parent genomes (\textsf{E} and  \textsf{F}) and a child genome (the recombinant,  \textsf{A}). In this case two independent recombinations share the same parent genomes, because the individuals containing the recombinants are siblings. Edge-annotation allows this complex genetic relationship to be fully encoded.
+% TODO - do we want to mention that by allowing multiple edges between nodes, we can 
```

Comment:
No - I think we'll have covered that in the previous section. PS we don't want to say "multiple edges between nodes" - it's just associating a *set* of intervals with an edge rather than one interval. Let's not get into the graph-vs-multigraph loop again.

Reply:
Yeah, I wasn't sure, hence the TODO. I think we do want to imply that GC etc can be captured here, but as you say, the main text should cover that.

Re graph/multigraph, I was only using the tskit terminology as a shorthand. "Multiple tskit edges between nodes" would have been more accurate, you are right.